CMakeLists.txt

```diff
@@ -33,6 +34,8 @@ option(CARE_ENABLE_MANAGED_PTR "Enable managed_ptr aliases, tests, and reproduce
 option(CARE_DISABLE_RAJAPLUGIN "Disable use of the RAJA plugin. WILL ALSO DISABLE MEMORY MOTION." OFF)
 option(CARE_ENABLE_EXTERN_INSTANTIATE "Enable extern instantiation of template functions" OFF)
 option(CARE_ENABLE_GPU_SIMULATION_MODE "Enable GPU simulation mode." OFF )
+cmake_dependent_option(CARE_ENABLE_LOOP_FUSER "Enable the loop fuser" OFF "ENABLE_CUDA OR ENABLE_HIP" OFF)
```

Comment:
Danny, please use the line I gave you. We want the loop fuser to default to on for the CUDA build, not the HIP build or any other build.

Reply:
Actually, maybe we shouldn't be using cmake_dependent_option. Looking at the documentation a bit more, I think this prevents the user from setting the option.

Reply:
Wouldn't your line though disable the option for HIP build? The previous email suggested that Peter wanted it available for HIP testing

Reply:
Ah, I see your new comment now, will revert.